### PR TITLE
Update to latest cdk library version. 

### PR DESCRIPTION
**Pull request by nilshoffmann**
Fixed some interface / object changes.

### CODE REVIEW

**Review by pcm32 — COMMENTED** [review]
Thanks for this update Nils! I only have one remark. It used to be that the SilentChemObjectBuilder had some advantage in terms of performance (I think they registered no notifiers, normally registered for UI purposes if not mistaken, although this was very long ago). Won't the change to non-silent objects generate a performance issue here? Performance is central for FASTLipid.

**Comment by nilshoffmann** [comment]
IsotopeFactory has been converted into an abstract class in CDK 2.0. 
The class extending this, apart from XMLIsotopeFactory, is the following one:
http://cdk.github.io/cdk/2.0/docs/api/org/openscience/cdk/config/Isotopes.html
This does only have the Isotopes.getInstance() method without parameters.

https://github.com/cdk/cdk/blob/8a5d83d785caa16f2786a6ddab7ed63154265953/base/core/src/main/java/org/openscience/cdk/config/Isotopes.java

This uses a specializes IIsotope implementation that bypasses the builder completely, so that should be quick enough :-).

File: src/main/java/uk/ac/ebi/lipidhome/fastlipid/structure/StaticSmilesGenerator.java [hunk 1]
```diff
@@ -2,21 +2,37 @@
...
-        return sg.createSMILES(mol);
+        try {
+            AtomContainerManipulator.percieveAtomTypesAndConfigureAtoms(mol);
+            CDKHydrogenAdder adder
+                    = CDKHydrogenAdder.getInstance(SilentChemObjectBuilder.getInstance());
+            adder.addImplicitHydrogens(mol);
+            AtomContainerManipulator.convertImplicitToExplicitHydrogens(mol);
+            return sg.create(mol);
+        } catch (CDKException ex) {
+            Logger.getLogger(StaticSmilesGenerator.class.getName()).log(Level.SEVERE, null, ex);
```
Comment (nilshoffmann) [inline src/main/java/uk/ac/ebi/lipidhome/fastlipid/structure/StaticSmilesGenerator.java]: Are you okay with the standard logger being used here?

**Comment by pcm32** [comment]
Thanks! Apologies for the delay.

**Pull request merged**